apps/sms/views/shared/test/unit/contacts_test.js

```diff
@@ -789,7 +789,8 @@ suite('Contacts', function(done) {
     // to the console
     test('({}, ...)', function(done) {
```

Comment:
nit: remove the comment above, it's no longer relevant. And it's not still clear why you haven't changed ```assert.ok``` to ```assert.isTrue```?